app/models/article.rb

```diff
@@ -1,6 +1,6 @@
 class Article < ApplicationRecord
   validates_presence_of :title, :lead, :content, :category
-  enum category: [:culture, :economy, :international, :lifestyle, :local, :sports]
+  enum category: [:culture, :economy, :international, :lifestyle, :sports]
```

Comment:
Why did we remove local as a category?

Reply:
because it doesn't correlate with what we are doing on the front end.